units/en/unit1/dummy-agent-library.mdx

```diff
@@ -286,40 +286,48 @@ The only values that should be in the "action" field are:
 get_weather: Get the current weather in a given location, args: {"location": {"type": "string"}}
 example use : 
 
-{{
+{
   "action": "get_weather",
   "action_input": {"location": "New York"}
-}}
+}
 
 ALWAYS use the following format:
 
-Question: the input question you must answer
-Thought: you should always think about one action to take. Only one action at a time in this format:
+Question: the input question you must answer  
+Thought: you should always think about one action to take. Only one action at a time in this format:  
 Action:
 
-$JSON_BLOB (inside markdown cell)
+    ```json
```

Comment:
The `$JSON_BLOB (inside markdown cell)` tells the model what in that when generating the action, it should have the form of a JSON. That line must be there in order to make it work.

You can also follow the notebook through [this link](https://huggingface.co/agents-course/notebooks/blob/main/dummy_agent_library.ipynb)